FAQ-LICENSING.md

```diff
@@ -0,0 +1,53 @@
+# JTS License FAQ
+
+JTS is dual-licensed under the [Eclipse Public License 1.0](https://www.eclipse.org/legal/epl-v10.html) ("EPL") 
+**or** the [Eclipse Distribution License 1.0](http://www.eclipse.org/org/documents/edl-v10.php) (a Revised BSD-style license). 
+More detail is provided in the [LICENSES](LICENSES.md) document.
+
+This FAQ clarifies the implications of the JTS software licensing.
+
+## Terminology
+
+* **Downstream projects** are codebases which link to JTS libraries
+  * e.g. GeoTools, GeoServer, etc.
+* **Derivative projects** are ports of JTS to other languages 
+  * e.g. GEOS, JSTS, etc.
+
+
+## Licensing Frequently Asked Questions
+
+**A1. What is the difference between EDL and Revised BSD?**
+
+  Revised BSD is a license *family* (see [Wikipedia entry](https://en.wikipedia.org/wiki/BSD_licenses#3-clause_license_.28.22BSD_License_2.0.22.2C_.22Revised_BSD_License.22.2C_.22New_BSD_License.22.2C_or_.22Modified_BSD_License.22.29)).
+  Revised BSD is also known as "BSD 2.0" or "3-clause BSD".  It is a less restrictive version of the original BSD license.
+  
+  EDL is the actual defined license.
+  It is called that because Eclipse is the distributing organization.
+
+**A2. JTS no longer uses LGPL.  What does this mean for downstream projects?**
+
+  JTS is now released under a dual license: EDL **or** EPL.  
+  Dual-licensing gives downstream projects more options:
+
+* The **EDL** is a Revised BSD-style license.
+  It is **more** permissive than the LGPL.  
+  This allows a wider variety of uses.  For instance, it permits iOS use of JTS for mobile development.
+  
+* The **EPL** provides a similar permissiveness to LGPL, with different wording around patents
+
+**A3. But I thought LGPL and EPL do not mix? How does this effect derivative projects like the LGPL GEOS project?**
+
+  An LGPL project (like GEOS) can use JTS under the terms of the EDL license, which is compatible with the LGPL license.
+
+  In the longer term, the EPL license is being revised to work with GPL / LGPL.  
+  This will allow derivative projects the choice of using JTS under the terms of either the EDL or EPL license.
```

Comment:
I think a mention should be given here about the choice of `distributing` (rather than `using`) derivative projects under the terms of the `GPL` or `LGPL`. Ie: can a derivative project be distributed under the {GPL2,LGPL2,GPL3,LGPL3) license ?

Reply:
Understood, @strk can you provide an example text with your comment (trying to gather examples for discussion).

Reply:
reading your note:

>  Distribution of derivative projects under the terms of  the LGPL or GPL licenses (versions 2 or 3) is permitted  by the JTS license.

To go back to the above comment you are asking to revise "is compatible" and make a longer sentence.  My understanding is there is not much difference between distributing an unmodified jar vs porting to another language - in both cases JTS is a "dependency" being used via BSD license.

Here is revised txt:

```
  An LGPL project (like GEOS or GeoTools) can use JTS as a dependency, under the terms of the EDL license. The three clauses (common to all BSD-style licenses) allows GEOS to port the work to C and distribute the resulting work. This is possible because the LGPL license used by GEOS contains additional restrictions (notably a requirement to share modifications to GEOS with end-users) that exceed what is required by JTS, but do not conflict with what is required by JTS.

For a greater discussion on open source licenses and the compatibility between BSD and LGPL expressed above we encourage you to read the [FSF](https://www.gnu.org/licenses/license-list.en.html) commentary. 
```

I have removed the point about EPL because we are not taking advantage optional clause at present.